cmd/ssh_usercert_gen/main.go

```diff
@@ -64,9 +65,9 @@ type AppConfigFile struct {
 }
 
 const (
-	AuthLevelNone     = -1
-	AuthLevelPassword = 0
-	AuthLevelU2F      = 1
+	AuthLevelNone     = 0
+	AuthLevelPassword = 1
+	AuthLevelU2F      = 2
```

Comment:
My plan is not to make them the same category. My current toughts are to have the following levels::
None
Password
Federated
OTP
u2f

And let the admin decide which is the lower level for token management acceptable. OTP will include vip and potentially a  local otp token. This week as I add the oath2 auth that level will appear.